apps/spruce/src/components/Banners/RepoTrackerBanner/index.tsx

```diff
@@ -20,35 +20,35 @@ import {
   USER_PROJECT_SETTINGS_PERMISSIONS,
   REPOTRACKER_ERROR,
 } from "gql/queries";
-import { PortalBanner } from "./PortalBanner";
+import { PortalBanner } from "../PortalBanner";
 
-interface RepotrackerBannerProps {
+interface RepoTrackerBannerProps {
   projectIdentifier: string;
 }
-export const RepotrackerBanner: React.FC<RepotrackerBannerProps> = ({
+export const RepoTrackerBanner: React.FC<RepoTrackerBannerProps> = ({
```

Comment:
The changes make sense, but I think we shouldn't change the capitalization with `Repotracker`. I would want to stay consistent with the spelling in [our docs](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-and-Distro-Settings#repotracker-settings)

And on the Project Settings page, we call it Repotracker too, introducing an inconsistency into the UI codebase

```suggestion
export const RepotrackerBanner: React.FC<RepotrackerBannerProps> = ({
```

Reply:
That makes sense to me good catch!